CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * More useful error messages for assertion failures in legacy `tests`
 * Digest auth does not attempt retries for invalid credentials/configuration. It will continue to retry for missing configuration.
 * Auth will maintain it's state across a collection run. Digest auth no longer needs to send 2 requests for every digest auth protected item in a collection
+* Added support for custom DNS lookup
```

Comment:
Changelog needs to have an entry for the breaking change in `restrictedAddresses`

Reply:
Also `README` needs to be updated.